MenuConfig.xml

```diff
@@ -9,6 +9,9 @@
   <MenuRule Destination="030 Overview" SourcePattern="Howto:^58 jQuery Components/00 DevExtreme jQuery Components" VisibleFor="jQuery" />
 
   <MenuRule Destination="050 Getting Started/50 Accessibility" SourcePattern="Howto:^Common/Accessibility" />
+
+  <MenuRule Destination="050 Getting Started/56 Integration Guides/" SourcePattern="Howto:^Common/Integration Guides/"/>
```

Comment:
```suggestion
  <MenuRule Destination="050 Getting Started/56 Integration Guides/" SourcePattern="Howto:^Common/Integration Guides/"/>
```
I thought we decided to move the topics out of the Getting Started section. Also please check that this section is not visible in jQuery.